api/src/main/java/edu/cornell/mannlib/vitro/webapp/i18n/TranslationConverter.java

```diff
@@ -327,4 +327,8 @@ private ChangeSet makeChangeSet(RDFService rdfService) {
 		return cs;
 	}
 	
+	private String getClassUri() {
+		return "java:" + this.getClass().getCanonicalName();
+	}
```

Comment:
so, here the information about the user who requested changes will be actually java class, right? For me it is ok to log that as well.

Reply:
Yes. Class uri.